src/main/scala/chisel3/util/experimental/BoringUtils.scala

```diff
@@ -45,8 +49,8 @@ class BoringUtilsException(message: String) extends Exception(message)
   *
   * ===Hierarchical Boring===
   *
-  * Hierarchcical boring involves connecting one sink instance to another source instance in a parent module. Below,
-  * module `Top` contains an instance of `Cosntant` and `Expect`. Using [[BoringUtils.bore]], we can connect
+  * Hierarchical boring involves connecting one sink instance to another source instance in a parent module. Below,
```

Comment:
```suggestion
  * Hierarchical boring involves driving a signal in a sink instance from a signal in a source instance, when both instances are in the same parent module. Below,
```

same question... what exactly is an instance? I might rephrase as above...